public/index.html

```diff
@@ -5,6 +5,8 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css' rel='stylesheet' />
```

Comment:
Do we want to user to download the stylesheet from the CDN on each load, or would we prefer to  bundle it up with the application? 

If you remove this, and enter the following in `app.js`, you'll find that the result should be the same:

```js
import 'mapbox-gl/dist/mapbox-gl.css';
```

Reply:
thanks for putting it here